### PR TITLE
Updated getCurrentVersion regex

### PR DESCRIPTION
**Pull request by stevenorell**
The existing regex wasn't matching two-digit tags (e.g. 4.7). New regex grabs everything after the last slash.

### CODE REVIEW

**Comment by wesleytodd** [comment]
Its been a while since I have looked at this code, but this has a glaring issue now that I review it.  A better implementation would be:

```
\d+\.\d+(\.\d+)?
```

This would catch things like `4.123.0`, `1.2` and `1.2.124`.  Which the old implementation did not.  Re-make this with that regex and I will merge/release.

**Comment by wesleytodd** [comment]
If you are interested, here is a handy tool to visualize regexes: https://regexper.com/#%5Cd%2B%5C.%5Cd%2B(%5C.%5Cd%2B)%3F

**Comment by stevenorell** [comment]
Cool, will do. Thanks!

**Comment by stevenorell** [comment]
Regex is updated.

File: lib/misc.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ wpMisc.getCurrentVersion = function(cb) {
 
 	// List the tags
 	git.listRemote('--tags ' + wpMisc.urls.gitRepo, function(err, tagsList) {
+
```
Comment (wesleytodd) [inline lib/misc.js]: Remove this extra line

Comment (stevenorell) [inline lib/misc.js]: Done and updated. Sorry for the delay.

**Comment by wesleytodd** [comment]
I will look at releasing this over the weekend, cant now because the tests fail and I am at work, so can't investigate.  But I will update when its released.

**Pull request merged**